concept/type/impl/TypeImpl.java

```diff
@@ -122,6 +122,10 @@ public boolean isAbstract() {
                 .flatMap(t -> graphMgr.data().getReadable(t.vertex)).map(thingConstructor);
     }
 
+    <THING> FunctionalIterator<THING> instancesExplicit(Function<ThingVertex, THING> thingConstructor) {
+        return graphMgr.data().getReadable(vertex).map(thingConstructor);
+    }
```

Comment:
helper method used everywhere to get explicit instances (see method above to see implementation of explicit + child instances)